### PR TITLE
Spam: make admin filters easier to understand

### PR DESCRIPTION
**Pull request by humitos**
Use a `gte` and `lt` applied to the spam filters so we show "_only_ projects
with score between N-M instead of >N"

### CODE REVIEW

File: readthedocs/projects/admin.py [hunk 1]
```diff
@@ -135,50 +136,79 @@ class ProjectSpamThreshold(admin.SimpleListFilter):
 
     def lookups(self, request, model_admin):
         return (
+            (
+                self.NOT_ENOUGH_SCORE,
+                _("Not enough score (1-{})").format(
```
Comment (agjohnson) [inline readthedocs/projects/admin.py:141]: "Not enough score" sounds like something other than "not spam". Any reason we don't just want to call these projects "not spam"?

```suggestion
                _("Not spam (1-{})").format(
```

**Review by agjohnson — COMMENTED** [review]
Makes sense, though it wasn't clear if the first level, "not enough score", was meant to describe legitimate projects or not.

File: readthedocs/projects/admin.py (hunk 1 above)
Comment (ericholscher) [reply readthedocs/projects/admin.py:141]: Agreed on this change 👍

**Review by ericholscher — APPROVED** [review]
This would be useful, I wanted this earlier today 👍

**Pull request merged**